tracing-subscriber/src/filter/layer_filters.rs

```diff
@@ -120,6 +120,8 @@ pub struct FilterFn<F = fn(&Metadata<'_>) -> bool> {
 /// [`Registry`]: crate::registry::Registry
 #[derive(Copy, Clone)]
 pub struct FilterId(u64);
+#[derive(Copy, Clone, Eq, PartialEq)]
+pub(crate) struct FilterMask(u64);
```

Comment:
Introducing this type without actually documenting or using it feels like cheating :p

Reply:
whoops, i started adding that but didn't finish --- i didn't mean to commit it yet! my bad